calico/etcdutils.py

```diff
@@ -96,9 +97,14 @@ class EtcdClientOwner(object):
     reconnecting, optionally copying the cluster ID.
     """
 
-    def __init__(self, etcd_authority):
+    def __init__(self, etcd_authority, etcd_scheme, etcd_key, etcd_cert,
```

Comment:
Surely better to add these new parameters as keyword args, then it isn't necessary to update all of the derived classes  unless you need to pass that info in.